docs/docs/release-notes/0.1.2.md

```diff
@@ -0,0 +1,197 @@
+ - sbt support by [@smarter] [\[28\]][28]
+ - DottyDoc by [@felixmulder] [\[29\]][29]
+ - Implicit Function Types by [@odersky] [\[30\]][30]
+ - Phantom Types by [@nicolasstucki] [\[31\]][31]
+ - Functions with more than 22 parameters by [@odersky] [\[32\]][32]
+ - Inline keyword by [@odersky] [\[33\]][33]
+
+[@odersky]: https://twitter.com/odersky
+[@DarkDimius]: https://twitter.com/DarkDimius
+[@smarter]: https://github.com/smarter
+[@felixmulder]: https://twitter.com/felixmulder
+[@nicolasstucki]: https://github.com/nicolasstucki
+[@liufengyun]: https://github.com/liufengyun
+[@OlivierBlanvillain]: https://github.com/OlivierBlanvillain
+[@olhotak]: https://plg.uwaterloo.ca/~olhotak/
+[@retronym]: https://github.com/retronym
+[@xeno-by]: https://github.com/xeno-by
+[@vsalvis]: https://github.com/vsalvis
+[@magarciaEPFL]: https://github.com/magarciaEPFL
+
+[1]: https://docs.google.com/document/d/1h3KUMxsSSjyze05VecJGQ5H2yh7fNADtIf3chD3_wr0/edit
+[2]: https://infoscience.epfl.ch/record/222780?ln=en
+[3]: http://dotty.epfl.ch/docs/reference/intersection-types.html
+[4]: http://dotty.epfl.ch/docs/reference/union-types.html
+[5]: http://dotty.epfl.ch/docs/reference/adts.html
+[6]: http://dotty.epfl.ch/docs/reference/desugarEnums.html
+[7]: http://dotty.epfl.ch/docs/reference/implicit-by-name-parameters.html
+[8]: https://infoscience.epfl.ch/record/228518
+[9]: http://docs.scala-lang.org/sips/pending/static-members.html
+[10]: http://docs.scala-lang.org/sips/pending/improved-lazy-val-initialization.html
+[11]: http://magarciaepfl.github.io/scala/
+[12]: https://github.com/lampepfl/dotty/commit/b2215ed23311b2c99ea638f9d7fcad9737dba588
+[13]: https://github.com/lampepfl/dotty/pull/187
+[14]: https://github.com/lampepfl/dotty/pull/217
+[15]: http://dotty.epfl.ch/docs/reference/trait-parameters.html
+[16]: https://github.com/lampepfl/dotty/commit/89540268e6c49fb92b9ca61249e46bb59981bf5a
+[17]: https://github.com/lampepfl/dotty/pull/174
+[18]: https://github.com/lampepfl/dotty/pull/488
+[19]: https://github.com/lampepfl/dotty/pull/174
+[20]: https://github.com/lampepfl/dotty/pull/411
+[21]: https://github.com/lampepfl/dotty/pull/1364
+[22]: https://github.com/lampepfl/dotty/pull/1227
+[23]: https://github.com/lampepfl/dotty/pull/117
+[24]: https://github.com/lampepfl/dotty/pull/2532
+[25]: https://github.com/lampepfl/dotty/pull/2194
+[26]: https://github.com/lampepfl/dotty/pull/213
+[27]: https://github.com/lampepfl/dotty/pull/2513
+[28]: https://github.com/lampepfl/dotty/pull/2361
+[29]: https://github.com/lampepfl/dotty/pull/1453
+[30]: http://dotty.epfl.ch/docs/reference/implicit-function-types.html
+[31]: https://github.com/lampepfl/dotty/pull/2136
+[32]: https://github.com/lampepfl/dotty/pull/1758
+[33]: http://dotty.epfl.ch/docs/reference/inline.html
+
+# Contributors
+Dotty team and contributes have closed 750 issues and has merged a total of 1258 pull requests.
```

Comment:
<ins>The</ins> Dotty team and contribut<ins>ors</ins> ... and <ins>have</ins> merged